operator/internal/central/reconciler/reconciler.go

```diff
@@ -34,8 +34,12 @@ func RegisterNewReconciler(mgr ctrl.Manager, selector string) error {
 		pkgReconciler.WithPreExtension(extensions.ReconcileScannerDBPasswordExtension(mgr.GetClient(), mgr.GetAPIReader())),
 		pkgReconciler.WithPreExtension(extensions.ReconcileScannerV4DBPasswordExtension(mgr.GetClient(), mgr.GetAPIReader())),
 		pkgReconciler.WithPreExtension(extensions.ReconcileAdminPasswordExtension(mgr.GetClient(), mgr.GetAPIReader())),
-		pkgReconciler.WithPreExtension(extensions.ReconcilePVCExtension(mgr.GetClient(), mgr.GetAPIReader(), extensions.PVCTargetCentral, extensions.DefaultCentralPVCName)),
-		pkgReconciler.WithPreExtension(extensions.ReconcilePVCExtension(mgr.GetClient(), mgr.GetAPIReader(), extensions.PVCTargetCentralDB, extensions.DefaultCentralDBPVCName)),
+		pkgReconciler.WithPreExtension(extensions.ReconcilePVCExtension(mgr.GetClient(), mgr.GetAPIReader(), extensions.PVCTargetCentral,
+			extensions.DefaultCentralPVCName, extensions.WithDefaultClaimSize(extensions.DefaultPVCSize))),
+		pkgReconciler.WithPreExtension(extensions.ReconcilePVCExtension(mgr.GetClient(), mgr.GetAPIReader(), extensions.PVCTargetCentralDB,
+			extensions.DefaultCentralDBPVCName, extensions.WithDefaultClaimSize(extensions.DefaultPVCSize))),
+		pkgReconciler.WithPreExtension(extensions.ReconcilePVCExtension(mgr.GetClient(), mgr.GetAPIReader(), extensions.PVCTargetCentralDBBackup,
+			extensions.DefaultCentralDBBackupPVCName, extensions.WithDefaultClaimSize(extensions.DefaultPVCSize))),
```

Comment:
Now I'm confused, why have an option in the first place if all uses specify the same size? 🤔 
IIRC one of these was supposed to differ?
If I got this right, please also make it such that not passing a `WithDefaultClaimSize` will cause `DefaultPVCSize` to be used and drop the option as appropriate..

Reply:
Makes sense, done.